docs/database-views/input4MIPs_datasets_CMIP7.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+<meta name="author" content="Paul J. Durack, Zebedee Nicholls" />
+<meta name="description" content="Controlled vocabulary for input4MIPs" />
+<meta name="keywords" content="HTML, CSS, JavaScript" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<link rel="stylesheet" type="text/css" charset="utf-8" href="https://pcmdi.github.io/assets/dataTables/jquery.dataTables.min.css" />
+<script type="text/javascript" charset="utf-8" src="https://pcmdi.github.io/assets/jquery/jquery.slim.min.js"></script>
+<script type="text/javascript" charset="utf-8" src="https://pcmdi.github.io/assets/dataTables/jquery.dataTables.min.js"></script>
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script type="text/javascript" src="https://pcmdi.github.io/assets/google/googleAnalyticsTag.js" ></script>
+<script type="text/javascript">
+//<![CDATA[
+$(document).ready( function () {
+    $('#table_id').DataTable({pageLength: 25});
+    } );
+//]]>
+</script>
+<title>input4MIPs CMIP7 datasets</title>
+</head>
+<body>
+<p><h1>input4MIPs CMIP7 datasets: v6.6.11a1</h1><p>
+<h4>
+<a href='/'>Home</a>
+| <a href='input4MIPs_delivery-summary.html'>Delivery summary view</a>
+| <a href='input4MIPs_source-id_CMIP7.html'>Source ID-level view</a>
```

Comment:
If the CMIP6Plus files have their identifier, may as well be explicit for CMIP7 too

Reply:
fair, will update.